app/components/Chessguessr.tsx

```diff
@@ -7,6 +7,7 @@ import Modal from "./Modal/Modal";
 import { Game } from "~/utils/types";
 import { useWindowSize } from "~/hooks/useWindowSize";
 import TutorialModal from "./TutorialModal";
+import { useHotkeys } from "react-hotkeys-hook";
```

Comment:
Small, open-source(MIT) and well-maintained library. No need to reinvent the wheel imo.

[Repository](https://github.com/JohannesKlauss/react-hotkeys-hook)